_posts/2024-05-31-How-to-connect-with-WildFly-community.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "How to Connect with the WildFly Community"
+date:   2024-05-31
+tags:   WildFly Community
+author: rchakrab
+description: Article about How to connect with the WildFly Community.
+---
+
+= How to Connect with the WildFly Community
+
+WildFly is an open-source, production-ready, flexible and lightweight application server, offers all the necessary features to run and deploy Java applications. Connecting with the WildFly community can provide valuable support, enhance your understanding of the server, and offer opportunities to contribute to the project. Here's how you can get involved:
+
+== Get an Idea about WildFly
+=== Explore the WildFly Website
+
+Visit WildFly's official https://www.wildfly.org/[website] to access comprehensive documentations, guides, downloads, and the latest news. The website serves as your primary resource for understanding and using WildFly effectively.
+
+=== Get Started with WildFly
+Begin by downloading https://www.wildfly.org/downloads/[WildFly] and following the https://www.wildfly.org/get-started/[getting started guide] on the website. This guide walks you through installation, initial configuration, and deploying a sample application, setting you up for success with your first WildFly server.
+
+=== Fork the WildFly Repository on GitHub and Play with It
+Explore WildFly's https://github.com/wildfly/wildfly[source code] by https://github.com/wildfly/wildfly/fork[forking] the repository on GitHub. Clone your forked repository to your local machine, experiment with the code, and understand the inner workings of WildFly. This hands-on approach can deepen your knowledge and potentially lead to contributions to the project. You can also go through the article on https://www.wildfly.org/news/2024/03/01/How-to-contribute-to-WildFly/[How to contribute to WildFly].
+
+== Join the WildFly Google Group and Mailing List
+=== Engage in the WildFly Forums
+Participate in the https://groups.google.com/g/wildfly[WildFly Google Group] to ask questions, share knowledge, and engage with other users and developers. The forum is a valuable platform for seeking help, providing answers, and staying updated with announcements and community discussions.You can also chat with the WildFly community members as well as with contributors via our official https://wildfly.zulipchat.com/[Zulip] chatroom.
```

Comment:
s/discussion.You/discussions. You/g

Reply:
I suggest adding something like:

"For discussions about using WildFly, try the https://wildfly.zulipchat.com/#narrow/stream/196266-wildfly-user[wildfly-user stream] in the chatroom.